### ISSUE TITLE
package install error

### ISSUE THREAD

**Issue by hanbaoSB** [jtbates/kliqfindr#3]
dvtools and "jsonlite" and rtools are all installsed

ERROR MESSAGES:

 > library(kliqfindr) Error in library(kliqfindr) : there is no package called ‘kliqfindr’ 
> devtools::install_github("jtbates/kliqfindr") WARNING: Rtools is required to build R packages, but the version of Rtools previously installed in C:/rtools40 has been deleted.  Please download and install Rtools custom from http://cran.r-project.org/bin/windows/Rtools/. Downloading GitHub repo jtbates/kliqfindr@HEAD Error: Failed to install 'kliqfindr' from GitHub:   Could not find tools necessary to compile a package Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.
--
 
> | >
>

**Comment by jtbates** [comment]
The error message says that you have deleted Rtools
```
the version of Rtools previously installed in C:/rtools40 has been deleted
```
Can you try reinstalling Rtools and then attempt kliqfindr install again?

**Comment by jtbates** [comment]
Does this work for you?
```
install.packages("jsonlite", type = "source")
```

**Comment by hanbaoSB** [comment]
yes. but not working with type = "source", but installed succesfully.
install.packages("jsonlite"

**Comment by hanbaoSB** [comment]
Like the followings:
install.packages('jsonlite', dependencies=TRUE, repos='http://cran.rstudio.com/')

**Comment by jtbates** [comment]
If you cannot install other packages from source then the problem isn't with the kliqfindr package. We don't have a binary release at this time, so you need to have a working Rtools install.

**Comment by hanbaoSB** [comment]
Ok, let me try again. Thanks!

**Comment by jtbates** [comment]
Perhaps the problem is related to this issue?
https://github.com/r-lib/pkgbuild/issues/92
